packages/contracts-periphery/deploy/drippie/Drippie.ts

```diff
@@ -0,0 +1,24 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+import { getDeployConfig } from '../../src'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deployer } = await hre.getNamedAccounts()
+
+  const config = getDeployConfig(hre.network.name)
```

Comment:
This pattern is great and I think something like this should be implemented via hardhat plugin so we can use it across all deployments